### ISSUE TITLE
Menu icon in Mate panel  is not announced by screen reader when focused

### ISSUE THREAD

**Issue by vojtapolasek** [ubuntu-mate/mate-menu#90]
I am running mate-menu on Fedora 42.
Exact package version is mate-menu-20.04.3-19.fc42.noarch

## Steps to reproduce:
1. add mate-menu to Mate panel as an applet
2. Turn on the Orca screen reader
3. Switch to the panel with ctrl+alt+tab
4. use TAB to travdre the panel and focus the mate-menu icon

## Result:
No feedback is announced by screen reader

## Expected result:
Reasonable description.

## Hints:
Try to add accessible name / description to the applet. Not sure how to help you more.
